web/tailwind.config.js

```diff
@@ -8,9 +8,47 @@ export default {
   theme: {
     extend: {
       fontFamily: {
-        sans: ['Montserrat'],
-        body: ['Open Sans'],
+        sans: ['Work Sans'],
+        lora: ['Lora', 'serif'],
+        poppins: ['Poppins', 'serif']
```

Comment:
Poppins is a sans serif font, so changing the fallback font to 'sans' would be clearer. 

Also thinking about changing lora to `serif: ['Lora', 'serif']`, in case the serif font changes from Poppins to another one. but then we also have the issue where Poppins is the only literal name. Might cause issues in flexibility later but for now probably ok.